cmd/validator/validator.go

```diff
@@ -388,6 +389,31 @@ func mainInit() int {
 	return exitStatus
 }
 
+func getExcludeFileTypes(configExcludeFileTypes string) []string {
+	excludeFileTypes := strings.Split(configExcludeFileTypes, ",")
+	uniqueFileTypes := misc.ArrToMap(excludeFileTypes...)
```

Comment:
I'm a bit worried by the implementation, but maybe it's already the case with extension exclusions.

My problem is the fact someone might for whatever reasons use JSON, Yaml, or even jSon as argument for file types exclusion

Reply:
Actually,  we lower excludeFileTypesPtr before validation, so the case you described is covered. But to be safe, I added toLower into getExcludeFileTypes and added tests for that.

https://github.com/Boeing/config-file-validator/blob/main/cmd/validator/validator.go#L159